_docs/index.md

```diff
@@ -10,9 +10,11 @@ type: markdown
 ---
 {% include home.html %}
 
-# Welcome
```

Comment:
Can we leave the title as Welcome? This matches the tab name, and matches what we do in all other sections like Concepts, Tasks, Reference. Maybe just:

# Welcome - Istio 0.1